src/dcqc/suites/suites.py

```diff
@@ -22,7 +22,7 @@ class JsonLdSuite(JsonSuite):
 
 class TiffSuite(FileSuite):
     file_type = FileType.get_file_type("TIFF")
-    add_tests = (tests.LibTiffInfoTest,)
+    add_tests = (tests.LibTiffInfoTest, tests.GrepDateTest)
```

Comment:
Just to confirm, there's a non-binary portion of TIFF files that can be grep'ed? If the intent is to scan the metadata portion of the TIFF file, then we can revisit this approach once we add support for scanning a portion of a remote file.

Reply:
Yes, @adamjtaylor confirmed that this works